### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by blazekadam**
- remove dev install section
- update install instructions to use pypi
- add license badge
- reshape badge to a one-liner
- add rethinkdb extension referrence

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,34 +17,31 @@ Please note that all operating systems are expected to be fully up-to-date with
...
 ## Extensions
 **cxflow** is meant to be extremely lightweight.
 For that reason the whole functionality is divided into various extensions with separate dependencies.
 
-## Officially Supported Extensions
+At the moment we support the following extensions:
 
 - [cxflow-tensorflow](https://github.com/Cognexa/cxflow-tensorflow) - TensorFlow support
 - [cxflow-scikit](https://github.com/Cognexa/cxflow-scikit) - scientific computations and statistics
+- [cxflow-rethinkdb](https://github.com/Cognexa/cxflow-rethinkdb) - rethinkDB hook for training management with noSQL (experimental)
```
Comment (FloopCZ) [inline README.md]: please capitalize RethinkDB and NoSQL

File: README.md [hunk 2]
```diff
@@ -19,34 +17,31 @@ Please note that all operating systems are expected to be fully up-to-date with
...
-
-### Standard Installation
-1. Install **cxflow** `$ pip install git+git@github.com:Cognexa/cxflow.git`
-
-### Development Installation
-1. Clone the **cxflow** repository `$ git clone git@github.com:Cognexa/cxflow.git`
-2. Enter the directory `$ cd cxflow`
-3. **Optional**: *Install some of the required of packages (e.g. TensorFlow).
-4. Install **cxflow**: `$ pip install -e .`
+**cxfow** is available at official PyPI repository; hence, the recommended installation is with `pip`:
```
Comment (FloopCZ) [inline README.md]: is available through the official

Comment (FloopCZ) [inline README.md]: `cxfow` -> `cxflow`

File: README.md [hunk 3]
```diff
@@ -19,34 +17,31 @@ Please note that all operating systems are expected to be fully up-to-date with
...
+pip install cxflow
+```
 
 ## Usage
 The installation process installs `cxflow` command which might be used simply from the command line.
 Please refer to repository [Cognexa/cxMNIST](https://github.com/Cognexa/cxMNIST) for more information.
 
-## Tutorials
-The following tutorials serve as a gentle introduction to the cxflow framework:
+## Docs & tutorials
```
Comment (FloopCZ) [inline README.md]: please capitalize Tutorials

File: README.md (hunk 2 above)
Comment (blazekadam) [inline README.md]: thanks, done

Comment (blazekadam) [inline README.md]: thanks, done

File: README.md (hunk 3 above)
Comment (blazekadam) [inline README.md]: thanks, done

File: README.md (hunk 1 above)
Comment (blazekadam) [inline README.md]: thanks, done

**Pull request merged**